src/media/css/elements/nav/index.styl

```diff
@@ -0,0 +1,212 @@
+@require('../../constants/typography');
+@require('constants');
+
+mkt-nav {
+  border-right: 2px solid rgba(0, 0, 0, 0.2);
+  fixed: top 0 left 0;
+  font-size: 18px;
+  height: 100%;
+  overflow-y: scroll;
+  transition: transform cubic-bezier(0.86, 0, 0.07, 1) 333ms;
+  width: 260px;
+  z-index: $zindex--nav-hamburger-root;
+
+  li {
+    margin: 25px 0;
+  }
+
+  a {
```

Comment:
Since selectors are interpreted from right to left each time an `<a>` or `<li>` is encountered the browser has to check if it has a `<mkt-nav>` as a parent. I'm not sure how costly this is but it seems rather inefficient.

If I were writing this I'd change this selector from `mkt-nav a` to `.mkt-nav--link`. Or maybe something more descriptive than `link` but I'm not sure what this is targeting exactly. That's still pretty low specificity, I think it would be the same as using the tag directly in practice.